org.eclipse.editorconfig.core/src/main/java/org/eclipse/editorconfig/internal/core/EditorConfigCorePlugin.java

```diff
@@ -0,0 +1,44 @@
+package org.eclipse.editorconfig.internal.core;

```

Comment:
What is the problem? Is it a problem with format? Which format do you use? Could you commit it?